docker-compose.prod.yml

```
@@ -0,0 +1,31 @@
+version: "3.9"
+
+services:
+  main:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

```suggestion
    build: .
```

`dockerfile: Dockerfile` なら context  指定だけでよかったはず

---

書いておいたほうが一応安全みたいな記事がどっかにあった気がする（受け売り）